### PR TITLE
Customer api beta

### PR DESCRIPTION
**Pull request by tatiana-perry**
This is the very beginning stages of the Customers Beta API. 

https://bigcommerce-customer-attributes-beta.docs.stoplight.io/

Still to do

- [ ] Get Questions answered
- [x] Add write up on differences between this and current along with a few workflows   
- [x] Fix Swagger Docs to reflect final changes
- [ ] Check final site before giving to partners and setting up any authentication on site if needed
- [x] Adding any required users to site backend for updates/changes

### CODE REVIEW

**Comment by tatiana-perry** [comment]
https://bigcommerce-customer-attributes-beta.docs.stoplight.io/

**Comment by tatiana-perry** [comment]
@karen-white sneak peek. Still working on it. Will get it done prob by the end of the day and have it reviewed for `technical correctness`. Then we can dig on the content.

**Comment by tatiana-perry** [comment]
@aleachjr 
Here is the first draft of the Customer Attributes
This includes my questions and a documentation draft. There is a section called Current Limitations that I left blank. Anything that I should call out there?

I also included the swagger doc. Mine is a bit different than yours. It splits the models up, includes examples and more descriptions. This makes it more UI friendly. 

[Documentation/Questions](https://github.com/bigcommerce-labs/dev-docs-beta/blob/3b2055a9c04192157dcf35339aa680cfd1a7b6e2/beta/customers/customers.md)

[Swagger Doc](https://github.com/bigcommerce-labs/dev-docs-beta/pull/5/files#diff-fc6ea0de627afc6809ec26d150dad88b)
[What the swagger doc looks like on the FE. ](https://bigcommerce-customer-attributes-beta.docs.stoplight.io/)

If possible, can you tag Derrek as well? I could not find his GitHub username.

**Comment by tatiana-perry** [comment]
@bookernath

File: beta/customers/customers.md [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+
+The name accepts a string. As a best practice, the name should be descriptive of the data it is associated with. 
+The type accepts a string and it can be set to string, number or date. 
+
+Note: Once the data type is set, it can not be changed. The attribute will need to be deleted then created again with the new data type. This will also delete it from the customer.
+
+### Create a Customer Attribute Value
+
+Creating a [customer attribute value](#link to PUT here) is done by creating the value and addig it to the customer in one call. 
+
```
Comment (tatiana-perry) [inline beta/customers/customers.md]: Need to add in examples

**Comment by bookernath** [comment]
@aleachjr do you have time to look at this?

**Comment by tatiana-perry** [comment]
Published

**Pull request closed without merging**